en/contributing/test.md

```diff
@@ -77,3 +87,138 @@ To run all telemetry tests:
 ```
 make && build/default/integration_tests_runner --gtest_filter="SitlTest.Telemetry*"
 ```
+
+## Writing Tests {#writing_tests}
```

Comment:
@julianoes So this section added to provide information about writing the tests. I am very tempted to move this to the plugins section, but it is about testing so it is relevant here. Thoughts?

Reply:
I would at least link to this from the plugins sections about how to add integration tests.

Reply:
Already done.